app/src/main/res/layout/exploration_fragment.xml

```diff
@@ -6,14 +6,15 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent">
 
-    <TextView
-      android:id="@+id/dummy_text_view"
-      android:layout_width="wrap_content"
+    <fragment
+      android:id="@+id/audio_fragment"
```

Comment:
This ID doesn't seem correct--shouldn't it be ``StateFragment``?

Also, maybe we should use a fragment transaction to add the fragment rather than using ``fragment`` tags. I don't have a strong opinion here, but I find it easier to reason about explicit fragment transactions rather than hiding them in the view inflation pipeline. In general, I think we should use explicit transactions except when there's no other option (e.g. an external library allows us to add views, and the view we need to add must be bound to a fragment).

Reply:
Actually, this process was taking time and I wanted to unlock @nikitamarysolomanpvt 

and given the case the even if we do not make this change then also its fine for now. I merged this PR without solving this issue.

I will create a separate pull request for this fix which @nikitamarysolomanpvt and @veena14cs